tekton/ci/jobs/tekton-github-tasks-completed.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: github-tasks-completed
+  namespace: tektonci
+  description: |
+    Verifies that a PR has all the tasks completed
+spec:
+  params:
+    - name: body
+      description: The body of the Pull Request
+  stepTemplate:
+    env:
+      - name: HOME
+        value: /tekton/home
+  steps:
+    - name: install-pyyaml
+      image: python:3-alpine
+      script: |
+        pip install pyyaml --user
```

Comment:
We should use a requirements.txt with hash checking enabled to prevent tampering (e.g. similar to go.mod + go.sum).

Reply:
similar to the comment below, doesn't seem like we handle any other ci jobs like that so maybe that should be part of a larger refactor of the plumbing?

Reply:
the same was discussed earlier that we need to create an image and address it more broadly outside of this PR